CHANGELOG.md

```diff
@@ -22,18 +22,19 @@ Friendly reminder: We have a [bug bounty program](https://hackerone.com/tendermi
 
 *January 19, 2021*
 
-This release includes a fix for a high-severity security vulnerability. 
-More information on this vulnerability will be released on January 26, 2021
-and this changelog will be updated. 
+This release includes a fix for a high-severity security vulnerability, 
+a DoS-vector that impacted Tendermint Core v0.34.0-v0.34.2. For more details, see
+[Security Advisory Mulberry](https://github.com/tendermint/tendermint/security/advisories/GHSA-p658-8693-mhvg) 
+or https://nvd.nist.gov/vuln/detail/CVE-2021-21271. 
 
-It also updates GoGo Protobuf to 1.3.2 in order to pick up the fix for
+Tendermint Core v0.34.3 also updates GoGo Protobuf to 1.3.2 in order to pick up the fix for
 https://nvd.nist.gov/vuln/detail/CVE-2021-3121. 
 
 Friendly reminder: We have a [bug bounty program](https://hackerone.com/tendermint).
 
 ### BUG FIXES
 
-- [evidence] [N/A] Use correct source of evidence time (@cmwaters)
+- [evidence] [[security fix]](https://github.com/tendermint/tendermint/security/advisories/GHSA-p658-8693-mhvg) Use correct source of evidence time (@cmwaters)
```

Comment:
```suggestion
- [evidence] [security fix](https://github.com/tendermint/tendermint/security/advisories/GHSA-p658-8693-mhvg) Use correct source of evidence time (@cmwaters)
```

Reply:
I actually want the brackets to show up, I think - it looks weird without them (look at the two commits on this PR to compare them).